docs/samples/custom/customized-urls/Readme.md

```diff
@@ -0,0 +1,106 @@
+# Customized URLs Sample
+
+In some situations, you may have a `custom model image` but can not modify its server routes, this server may also have one or more predict endpoints. Here is a simple sample to show how to deploy this custom service.
+
+## A simple sample 
+
+* `app.py` is a simple flask server, it has two endpoints: `customized_urls_test` and `hello_world`
+
+```python
+import os
+
+from flask import Flask
+
+app = Flask(__name__)
+@app.route('/customized/urls/demo/test/1')
+def customized_urls_test():
+    target = os.environ.get('TARGET', 'World')
+    return 'Hello {}. Func customized_urls_test is called!\n'.format(target)
+
+@app.route('/v1/models/customized-sample:predict')
+def hello_world():
+    target = os.environ.get('TARGET', 'World')
+    return 'Hello {}!\n'.format(target)
+
+if __name__ == "__main__":
+    app.run(debug=True,host='0.0.0.0',port=int(os.environ.get('PORT', 8080)))
+```
+
+* Build custom image.
+
+  Run the build command to build your image. Or you can use the the prebuild image:`iamlovingit/customized-urls:latest` 
+
+  ```shell
+  docker build -t ${your-dockerhub-id}/customized-urls:latest .
+  docker push ${your-dockerhub-id}/customized-urls:latest
+  ```
+
+* Config the Yaml
+
+```yaml
+apiVersion: serving.kubeflow.org/v1alpha2
+  kind: InferenceService
+  metadata:
+    labels:
+      controller-tools.k8s.io: "1.0"
+    name: customized-urls-sample
+  spec:
+    default:
+      predictor:
+        custom:
+          container:
+            name: hello
+            image: ${your-dockerhub-id}/customized-urls:latest
+```
+  
+* Deploy the  yaml and check the Inference service status
+
+```shel
+  kubectl apply -f customized-urls.yaml
+  kubectl get inferenceservice customized-urls-sample
```

Comment:
move this command out and create new item 

Check the `InferenceService` status